app/assets/javascripts/hq/dashboard.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
I don't see the point of adding a `.coffee` file that we are not using it at the moment. My opinion is to added only when  we *really* need it.